Doc/whatsnew/3.12.rst

```diff
@@ -576,19 +576,10 @@ asyncio
   :class:`asyncio.ThreadedChildWatcher`.
   (Contributed by Kumar Aditya in :gh:`98024`.)
 
-* The child watcher classes :class:`asyncio.MultiLoopChildWatcher`,
-  :class:`asyncio.FastChildWatcher`, :class:`asyncio.AbstractChildWatcher`
-  and :class:`asyncio.SafeChildWatcher` are deprecated and
-  will be removed in Python 3.14. It is recommended to not manually
-  configure a child watcher as the event loop now uses the best available
-  child watcher for each platform (:class:`asyncio.PidfdChildWatcher`
-  if supported and :class:`asyncio.ThreadedChildWatcher` otherwise).
-  (Contributed by Kumar Aditya in :gh:`94597`.)
-
-* :func:`asyncio.set_child_watcher`, :func:`asyncio.get_child_watcher`,
-  :meth:`asyncio.AbstractEventLoopPolicy.set_child_watcher` and
-  :meth:`asyncio.AbstractEventLoopPolicy.get_child_watcher` are deprecated
-  and will be removed in Python 3.14.
```

Comment:
``asyncio`` had two deprecations noted in "Improved Modules", these have been moved